.gitmodules

```diff
@@ -7,3 +7,6 @@
 [submodule "examples/lora/RadioLib"]
 	path = examples/lora/RadioLib
 	url = https://github.com/jgromes/RadioLib.git
+[submodule "ot-tock/openthread"]
+	path = ot-tock/openthread
+	url = https://github.com/openthread/openthread.git
```

Comment:
What ensures you have the git submodule checked out?

Reply:
Should we add `git submodule update --init' to the openthread build script `create_ot-tock`?

Reply:
I should be able to type `make` in an app and everything should just work.

Reply:
The scripts are updated now such that you should be able to do this.